components/kanban_tool/sources/new-board-collaborator/new-board-collaborator.mjs

```diff
@@ -0,0 +1,56 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "kanban_tool-new-board-collaborator",
+  name: "New Board Collaborator Created Event",
+  description: "Emit new events when a new board collaborator is created on selected board. [See the docs](https://kanbantool.com/developer/api-v3#fetching-boards-details)",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    boardId: {
+      propDefinition: [
+        common.props.app,
+        "boardId",
+      ],
+    },
+  },
+  methods: {
+    ...common.methods,
+    getResourceFnConfig() {
+      return {
+        resourceFn: this.app.getBoard,
+        resourceKey: "collaborators",
+        resourceFnArgs: {
+          boardId: this.boardId,
+        },
+        pagingCfg: {
+          noPaging: true,
+        },
+      };
+    },
+    getComparable() {
+      return new Date().getTime();
+    },
```

Comment:
Should this use the `created_at` time?
```suggestion
    getComparable(item) {
      return new Date(item?.created_at).getTime();
    },
```

Reply:
Actually no. It is because this and the next one uses id comparisons. This one is useless but left here because all of other sources has comparisons and those are done in common file.